### PR TITLE
Update Readme for external references, tutorials

### PR DESCRIPTION
**Pull request by r4bit999**
Added references for examples usage of Netzob, Tutorials and Detailed Documentation

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -45,6 +45,21 @@ General Information
 :Mailing list: Two lists are available, use the `SYMPA web interface <https://lists.netzob.org/wws>`_ to register.
 :IRC: You can hang-out with us on Freenode's IRC channel #netzob @ freenode.org.
 :Twitter: Follow Netzob's official accounts (@Netzob)
+Examples usage of Netzob, Tutorials and Detaild Documentation
```
Comment (gbossert) [inline README.rst]:
```suggestion
Examples usages of Netzob, Tutorials and Detailed Documentation
```

File: README.rst [hunk 2]
```diff
@@ -45,6 +45,21 @@ General Information
...
+----------------
+**Backup of www.netzob.org**
+
+* http://web.archive.org/web/2016*/www.netzob.org  
+
+**Detailed Netzob Documentation**
+
+* https://gdr-securite.irisa.fr/wp-content/uploads/sujet17-amossys.pdf  
+
+**Example usage of Netzob Library**
```
Comment (gbossert) [inline README.rst]:
```suggestion
**Examples usages of Netzob Library**
```

**Comment by r4bit999** [comment]
Ah right, next or next-1.1.dev0 ?

**Comment by gbossert** [comment]
On the `next` branch.

**Pull request merged**